test/suite.sh

```diff
@@ -37,8 +40,10 @@ assert "node_modules/.bin/browserify --debug test/sourcemaps-console.js | node b
 assert "node_modules/.bin/browserify --debug test/sourcemaps-uncaught.js | node bin/cmd.js | head -n 2" "Error: oups
       at test/sourcemaps-uncaught.js:2"
 
-PHANTOM=`which phantomjs`
-NODE=`which node`
+assert_end debugging
+
+PHANTOM=$(which phantomjs)
+NODE=$(which node)
```

Comment:
Well the biggest advantage I know is that you can use it anywhere. Like you can `$(nest $(commands $(deeply)))` without any problems. You can't nest back ticks. Also back ticks are deprecated.